src/Fixer/Basic/TrailingCommaInMultilineFixer.php

```diff
@@ -12,15 +12,14 @@
  * with this source code in the file LICENSE.
  */
 
-namespace PhpCsFixer\Fixer\ControlStructure;
+namespace PhpCsFixer\Fixer\Basic;
```

Comment:
The class is not marked as internal, isn't this a BC break?

Reply:
Do we hold a BC promise on any implementation of a single non-abstract rule?

Reply:
TBH I don't know 😅️

@keradus WDYT?

Reply:
I would name it BC break, following the rules I was following so far (no `@internal`, then under BC promise).

Reply:
I move it back. The reason was that this fixer has nothing to do with `ControlStructure` IMHO.